ui/tests/integration/components/date-dropdown-test.js

```diff
@@ -43,7 +43,8 @@ module('Integration | Component | date-dropdown', function (hooks) {
     assert.dom('[data-test-date-dropdown-submit]').hasText('Save', 'button renders passed in text');
   });
 
-  test('it renders dropdown and selects month and year', async function (assert) {
+  // skip until https://github.com/hashicorp/vault/pull/17575 is merged which refactors these tests and fixes flakiness
```

Comment:
Since this test has been flaky, opted to skip for now since this component and file were renovated quite a bit 🛠️ 

<img width="874" alt="Screen Shot 2023-01-03 at 1 46 22 PM" src="https://user-images.githubusercontent.com/68122737/210446640-84034bdc-0574-4a59-b6d5-68ecd27bcb03.png">